cromulent/data/linked-art.json

```diff
@@ -1,747 +1,528 @@
 {
   "@context": {
     "@version": 1.1,
-
     "crm": "http://www.cidoc-crm.org/cidoc-crm/",
-    "geo": "http://www.ics.forth.gr/isl/CRMgeo/",
```

Comment:
This file used to be able to be auto-generated, but changes in JSON-LD 1.1 to make the partitioning model easier make the auto-generation much harder (and thus is not up to date).  Can you remove this file from the PR please?